tests/example_kem.c

```diff
@@ -42,7 +42,7 @@ static OQS_STATUS example_stack(void) {
 #ifndef OQS_ENABLE_KEM_frodokem_640_aes // if FrodoKEM-640-AES was not enabled at compile-time
 	printf("[example_stack] OQS_KEM_frodokem_640_aes was not enabled at "
 	       "compile-time.\n");
-	return OQS_ERROR;
+	return OQS_SUCCESS; // nothing done successfully ;-)
```

Comment:
Was there a reason you changed this?  I would think we'd want these to return failures so that we know the test wasn't configured properly.

Reply:
Yes. Only with this change the test passes OK when Frodo640AES is not configured in (for "STD" and "NIST_R4" configs). The logic for this change was that if Frodo is not active there's no error in the library/configuration or the test. It's simply that Frodo is not there.

The alternative would be to re-write the whole test to "dynamically" select an algorithm that is available (e.g., by introducing yet another C-level config variable -- that could be dependent on the new `OQS_ALGS_ENABLED` `cmake`-level define). This would change the focus in this example code  from the simple "this is how to use one specific algorithm" to "this is how to configure liboqs for use with different algorithms", though: Would that be OK from your perspective?

Reply:
Ah, I just noticed this was an example file, not a test file.  And as far as I can tell none of the test suites actually execute this, so it's fine.  Although at this point it would probably be appropriate to switch it over to Kyber -- which I have done in a new commit.

Reply:
> And as far as I can tell none of the test suites actually execute this, so it's fine.

Not quite right: 
The examples _are_ part of testing: https://github.com/open-quantum-safe/liboqs/blob/a7e1b8a32f482f3b93b7d08d686d62942e5674df/tests/test_cmdline.py#L8-L13

> it would probably be appropriate to switch it over to Kyber -- which I have done in a new commit.

Thanks -- makes sense from a "standardization point of view" (and resolves the issue for config option "STD").

But this doesn't resolve it for config "NIST_R4" -- which does not include Kyber: CI would have failed if you had reverted the return code change that started the whole discussion (to fail on alg not available).

So three options now:
1) leave as-is now
2) make example fail on alg-not-available and add Kyber to "NIST_R4"
3)  make example fail on alg-not-available and create mechanism to select available algorithm (e.g., by new "OQS_ALGS_ENABLED-derived" C-level define as suggested above)